openpype/hosts/maya/api/lib.py

```diff
@@ -2148,6 +2162,26 @@ def get_frame_range():
     handle_start = asset["data"].get("handleStart") or 0
     handle_end = asset["data"].get("handleEnd") or 0
 
+    animation_start = frame_start
+    animation_end = frame_end
+
+    include_handles = include_handles_settings["include_handles_default"]
+    for item in include_handles_settings["per_task_type"]:
+        if current_task["type"] in item["task_type"]:
+            include_handles = item["include_handles"]
+            break
+    if include_handles:
+        animation_start -= int(handle_start)
+        animation_end += int(handle_end)
+
+    cmds.playbackOptions(
+        minTime=frame_start,
+        maxTime=frame_end,
+        animationStartTime=animation_start,
+        animationEndTime=animation_end
+    )
+    cmds.currentTime(frame_start)
```

Comment:
**🚨  Oops!**

Ok. So this PR implemented this logic here - but that's wrong. This shouldn't be in `get_frame_range()` but in `reset_frame_range()`.

The `get_frame_range()` function should not be adjusting the scene state.

---

@friquette @tokejepsen

Reply:
Yeah needs to be fixed asap. Good that we had 3 reviews on this :D right?